src/Tutor.Core/UseCases/Management/Stakeholders/StakeholderService.cs

```diff
@@ -54,15 +57,15 @@ public Result<T> Archive(int id, bool archive)
         return Result.Ok(stakeholder);
     }
 
-    public override Result<T> Update(T entity)
+    public Result<T> Update(T entity, string username)
```

Comment:
This "special case" seems like a code smell stemming from the Stakeholder hierarchy. Not a trivial thing to fix so we can leave it for now, but we should pay attention to this part of the code down the line and maybe improve it for the summer redesign.

Reply:
Agreed on the hierarchy problems. Should this change to update logic be included in the PR?
If not, updating learner will changes its credentials from index to email.

Reply:
We could if the hierarchy for Learner/LearnerCommercial were separate from Instructor which does not have the index field.